<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugInfo</class>
 <widget class="QWidget" name="DebugInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Found</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Functions</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Definitions</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="functionsText"/>
          </item>
          <item row="1" column="1">
           <widget class="QTextBrowser" name="definitionsText"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="functionCountLabel">
            <property name="text">
             <string>Total: 0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="definitionCountText">
            <property name="text">
             <string>Total: 0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Skipped</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Functions</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Definitions</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="skippedFunctionsText"/>
          </item>
          <item row="1" column="1">
           <widget class="QTextBrowser" name="skippedDefinitionsText"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="skippedDefinitionsCountText">
            <property name="text">
             <string>Total: 0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="skippedFunctionsCountLabel">
            <property name="text">
             <string>Total: 0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
